core/fortran-example/advanced/02_push/cabana_cpp_interface.cpp

```diff
@@ -0,0 +1,148 @@
+#include <Cabana_Parallel.hpp>
```

Comment:
Is this file repeated again in all of the `advanced/` directories? If so we should only have 1 copy